src/components/CreateNoteView/CreateNoteView.jsx

```diff
@@ -0,0 +1,148 @@
+import { Button, Grid, IconButton, TextField, Typography } from "@mui/material";
+import { Box } from "@mui/system";
+import { useFormik } from "formik";
+import CloseIcon from "@mui/icons-material/Close";
+import { useSnackbar } from "notistack";
+import PropTypes from "prop-types";
+
+import { createNoteFormValidationScema } from "validations";
+import notesApi from "api/notes";
+
+const CreateNoteHeader = () => (
+  <Box p={1} pb={2} sx={{ color: "#0000008a" }}>
+    <Typography variant="h5" align="left">
+      Add New Note
+    </Typography>
+  </Box>
+);
+
+const CreateNoteForm = ({ onSubmit }) => {
+  const formik = useFormik({
+    initialValues: {
+      noteTitle: "",
+      noteDescription: "",
+    },
+    validationSchema: createNoteFormValidationScema,
+    onSubmit: onSubmit,
+  });
+
+  return (
+    <div>
+      <form onSubmit={formik.handleSubmit}>
+        <TextField
+          fullWidth
+          id="noteTitle"
+          name="noteTitle"
+          label="Note Title"
+          sx={{ marginBottom: "20px" }}
+          value={formik.values.noteTitle}
+          onChange={formik.handleChange}
+          error={formik.touched.noteTitle && Boolean(formik.errors.noteTitle)}
+          helperText={formik.touched.noteTitle && formik.errors.noteTitle}
+        />
+        <TextField
+          fullWidth
+          multiline
+          rows={8}
+          id="noteDescription"
+          name="noteDescription"
+          label="Note Description"
+          sx={{ marginBottom: "20px" }}
+          value={formik.values.noteDescription}
+          onChange={formik.handleChange}
+          error={
+            formik.touched.noteDescription &&
+            Boolean(formik.errors.noteDescription)
+          }
+          helperText={
+            formik.touched.noteDescription && formik.errors.noteDescription
+          }
+        />
+        <Button
+          color="primary"
+          variant="contained"
+          type="submit"
+          sx={{ marginRight: "10px" }}
+        >
+          Submit
+        </Button>
+        <Button color="primary" variant="contained" type="reset">
+          Cancel
+        </Button>
+      </form>
+    </div>
+  );
+};
+
+const CreateNoteView = ({ onNoteAdd }) => {
```

Comment:
Only one component per file! Or small additional part accepted. But not 2 huge components. Move one of them to a separate file